run_linux.go

```diff
@@ -91,11 +91,7 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 		return err
 	}
 
-	defaultContainerConfig, err := config.Default()
-	if err != nil {
-		return errors.Wrapf(err, "failed to get container config")
-	}
-	b.configureEnvironment(g, options, defaultContainerConfig.Containers.Env)
+	b.configureEnvironment(g, options, []string{"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"})
```

Comment:
Could you add a comment here explaining why we're not using the config.Default here?  Just to avoid someone changing it back one early morning a year from now....

Reply:
Added `hardwire the environment to match docker build to avoid subtle and hard-to-debug differences due to containers.conf` above this.  Let me know if you'd prefer a different phrasing.

Reply:
What if the user wants to have these environment settings to be applied.

COMPANY="Dan Walsh Inc"
I want all of my images to have this environment variable set.

Reply:
Do you mean wanting that value included in the image configuration blob?  containers.conf doesn't do that, but there's an ENV instruction for that.  Otherwise, configuration from containers.conf is still used when the image we create is run by something like `podman run`.